var/spack/repos/builtin/packages/perl/package.py

```diff
@@ -130,10 +147,6 @@ def install_cpanm(self):
                 make()
                 make('install')
 
-    def setup_environment(self, spack_env, run_env):
-        """Set PERL5LIB to support activation of Perl packages"""
-        run_env.set('PERL5LIB', join_path(self.prefix, 'lib', 'perl5'))
```

Comment:
Do we need to keep this around to allow people to use Perl packages without using `spack activate`?